crates/router/src/lib.rs

```diff
@@ -124,9 +115,8 @@ pub async fn start_olap_server(conf: settings::Settings) -> ApplicationResult<(S
     Ok((server, app_state))
 }
 
-pub fn mk_oltp_app(
-    state: AppState,
-    request_body_limit: usize,
+pub fn get_application_builder(
+    json_cfg: JsonConfig,
```

Comment:
I think we can construct the `JsonConfig` instance within this function itself, instead of providing it as a parameter, considering that the instance is the same for both the apps.

Reply:
Done